examples/next/faust-nx/pages/_app.tsx

```diff
@@ -1,12 +1,11 @@
 import 'faustnx.config';
 import React from 'react';
-import client from 'client';
 import { FaustNXProvider } from 'faust-nx';
 import type { AppProps } from 'next/app';
 
 export default function MyApp({ Component, pageProps }: AppProps) {
   return (
-    <FaustNXProvider client={client} pageProps={pageProps}>
```

Comment:
I think we should still allow the user to supply their own client here irrespectively of the plugin system.

Reply:
With the `apolloClientOptions` filter hook, they could still supply their own client that operates within the bounds of the plugin system, e.g: 

```ts
class FaustNXApolloClient {
  apply({addFilter}) {
    addFilter('apolloClientOptions', 'faust-nx', (options, ctx) => {
      return {
        ssrMode: typeof window === 'undefined',
        connectToDevTools: typeof window !== 'undefined',
        link: new HttpLink({
          uri: process.env.NEXT_PUBLIC_GRAPHQL_ENDPOINT,
          fetchOptions: {
            method: 'POST',
          },
        }),
        cache: new InMemoryCache({
          possibleTypes,
          typePolicies: {
            RootQuery: {
              queryType: true,
            },
            RootMutation: {
              mutationType: true,
            },
          },
        }),
      }
    })
  }
}
```

Do you think this is a suitable altnerative?

Reply:
Sure thats fine